components/patient-measurements/ui/src/main/resources/PhenoTips/MeasurementSetDeletionService.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see http://www.gnu.org/licenses/
+-->
+
+<xwikidoc version="1.1">
+  <web>PhenoTips</web>
+  <name>MeasurementSetDeletionService</name>
+  <language/>
+  <defaultLanguage/>
+  <translation>0</translation>
+  <creator>xwiki:XWiki.Admin</creator>
+  <creationDate>1443818466000</creationDate>
+  <parent>PhenoTips.WebHome</parent>
+  <author>xwiki:XWiki.Admin</author>
+  <contentAuthor>xwiki:XWiki.Admin</contentAuthor>
+  <date>1443823387000</date>
+  <contentUpdateDate>1443823387000</contentUpdateDate>
+  <version>1.1</version>
+  <title/>
+  <comment/>
+  <minorEdit>false</minorEdit>
+  <syntaxId>xwiki/2.1</syntaxId>
+  <hidden>false</hidden>
+  <content>{{velocity}}
+    {{html clean="false"}}
+    #if ($xcontext.action == 'get')
+    #set ($docName = $request.document)
+    #set ($age = $request.age)
+    $response.setContentType('application/json')
+    #if ("$!docName" == "" || "$!age" == "")
+    $jsontool.serialize({"error": "Empty age or document name"})
+    $response.setStatus(400)
+    #break
+    #end
+    #set ($thatDoc = $xwiki.getDocument($docName))
+    #if (!$thatDoc)
+    $jsontool.serialize({"error": "Cannot load requested document"})
+    $response.setStatus(400)
+    #break
+    #end
+    #set ($objs = $thatDoc.getObjects("PhenoTips.MeasurementClass", "age", $age))
+    #foreach($obj in $objs)
+    #set ($discard = $thatDoc.removeObject($obj))
+    #end
+    #set ($discard = $thatDoc.save("Removed measurement set"))
+    $response.setStatus(200)
+    #end##
+    {{/html}}
```

Comment:
I think @danielpgross was trying to avoid the generation of wiki markup. The other option is to use the parameter `wiki="false"` (works for both `{{velocity}}` and `{{html}}` macros. Also `{{html}}` is unnecessary here.